descriptors/ModuleDescriptor-template.json

```diff
@@ -29,6 +29,23 @@
           "permissionsRequired": ["circulation-bff.requests.search-instances.get"],
           "modulePermissions": ["search.instances.collection.get"]
         },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation-bff/requests",
```

Comment:
Let's increase the minor version of the interface since we're adding an endpoint